txp/Pos/Script.hs

```diff
@@ -75,6 +76,9 @@ parseValidator t = do
         scrScript = Bi.serialize' scr,
         scrVersion = 0 }
 
+setVersionOne :: Bi Script_v0 => Script -> Script
```

Comment:
Just derive lenses for `Script` and use them here instead of doing this.

Reply:
No, wait, actually it's even simpler. Why do you need `setVersionOne` at all? Just write `intValidatorWithVersionOne = intValidator {scrVersion = 1}` at the call site.

Reply:
Hm, strange, I am quite sure it was used in two places earlier, so I asked to move it into a separate function to avoid (minor) duplication. Also moving it into a separate function allowed to write `setVersionOne <$> somethingBig`, but now it's not needed.